aiida/backends/tests/test_caching_config.py

```diff
@@ -37,7 +38,7 @@ def setUp(self):
                 'disabled': ['aiida.orm.calculation.job.simpleplugins.templatereplacer.TemplatereplacerCalculation', 'aiida.orm.data.bool.Bool']
             }
         }
-        with tempfile.NamedTemporaryFile() as tf, open(tf.name, 'w') as of:
+        with tempfile.NamedTemporaryFile() as tf, io.open(tf.name, 'w', encoding='utf8') as of:
```

Comment:
this looks wrong: `tempfile.NamedTemporyFile` opens the file already and provides `mode` and `encoding`attributes, so I don't get why we reopen the file again

Reply:
I agree - it's redundant. Removed the io.open call.